webpack.config.js

```diff
@@ -35,14 +35,6 @@ module.exports = {
         ],
         loader: 'babel'
       },
-       {
-        test: /\.glsl$/,
```

Comment:
AIUI, this is no longer need because it's now taken care of by the `babel-plugin-webpack-loaders`, but the README for that project says that it shouldn't be used during a Webpack compilation: https://github.com/istarkov/babel-plugin-webpack-loaders#warning (that plugin runs Webpack synchronously during the Babel transformation, which is itself run by Webpack for bundling)

Maybe it'd be better to have different Babel configurations depending on the usage: when calling babel-cli to produce CommonJS, then use `.babelrc` that calls the babel-plugin-webpack-loader with the babel-webpack.config.js; and when calling webpack, use the raw-loader in webpack.config.js and a different babel config that does not use the babel-plugin-webpack-loader (by inlining the babel config in the babel-loader's `query` and setting `babelrc: false`).